cells/xschem/xschemrc

```diff
@@ -296,6 +300,13 @@ lappend tcl_files ${XSCHEM_SHAREDIR}/ngspice_backannotate.tcl
 set toolbar_visible 1
 # set toolbar_horiz   1
 
+###########################################################################
+#### WEB URL DOWNLOAD HELPER APPLICATION
+###########################################################################
+#### used to download files from web: default: {curl -f -s -O}
```

Comment:
What is the use case for these web based macros?

Reply:
This allows xschem to open a design hosted -for example- on github directly:
xschem https://raw.githubusercontent.com/ceciocauis/ceciocauis/main/IO-cells/xschem/BW_tb.sch
I found this useful to quickly view designs without cloning whole repositories.

Reply:
Cool! Neat feature